Resources/config/services.yml

```diff
@@ -1,11 +1,28 @@
 services:
   werkspot.bing_ads_api_client:
```

Comment:
please use 4 spaces for indentation like we do for our code and rest of our projects